sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2370,4 +2370,17 @@ class HiveDDLSuite
       ))
     }
   }
+
+  test("SPARK-25464 create a database with a non empty location") {
```

Comment:
Do we have a test case to check "create a database with an empty location"?

Reply:
yes  [Test code is here](https://github.com/apache/spark/blob/master/sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala#L1119)